packages/datepickers/src/styled/StyledMenu.tsx

```diff
@@ -6,80 +6,110 @@
-    return `margin-left: ${marginAmount};`;
+  let animationKey;
+
+  if (props.placement.startsWith('top')) {
+    animationKey = 'bottom';
+  } else if (props.placement.startsWith('right')) {
+    animationKey = 'left';
+  } else if (props.placement.startsWith('bottom')) {
+    animationKey = 'top';
+  } else if (props.placement.startsWith('left')) {
+    animationKey = 'right';
   }
 
-  return '';
+  const animation = keyframes`
+    /* stylelint-disable property-no-unknown */
+    0% {
+      margin-${animationKey}: -${props.theme.space.base * 5}px;
+    }
+
+    100% {
+      margin-${animationKey}: 0;
+    }
+    /* stylelint-enable property-no-unknown */
+  `;
+
+  return css`
+    animation: ${animation} 0.2s cubic-bezier(0.15, 0.85, 0.35, 1.2);
+  `;
 };
 
-interface IStyledMenuViewProps extends HTMLAttributes<HTMLUListElement> {
+interface IStyledMenuViewProps {
   isCompact?: boolean;
   placement?: POPPER_PLACEMENT;
-  animate?: boolean;
+  isAnimated?: boolean;
 }
 
 /**
- * Accepts all `<ul>` props
+ * 1. Positioned relative to controlling item.
+ * 2. Opt out of browser default list margin.
+ * 3. Prevent controlling item cursor inheritance.
+ * 4. Opt out of browser default list padding.
+ * 5. Prevent controlling item whitespace inheritance.
  */
-const StyledMenuView = styled.ul.attrs<IStyledMenuViewProps>(props => ({
+const StyledMenuView = styled.div.attrs<IStyledMenuViewProps>(props => ({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION,
-  className: classNames(MenuStyles['c-menu'], {
-    // Size
-    [MenuStyles['c-menu--sm']]: props.isCompact,
-
-    // Placement
-    [MenuStyles['c-menu--up']]:
-      props.placement === 'top' || props.placement === 'top-start' || props.placement === 'top-end',
-    [MenuStyles['c-menu--right']]:
-      props.placement === 'right' ||
-      props.placement === 'right-start' ||
-      props.placement === 'right-end',
-    [MenuStyles['c-menu--left']]:
-      props.placement === 'left' ||
-      props.placement === 'left-start' ||
-      props.placement === 'left-end',
-    [MenuStyles['c-menu--down']]:
-      props.placement === 'bottom' ||
-      props.placement === 'bottom-start' ||
-      props.placement === 'bottom-end',
-
-    // State
-    [MenuStyles['is-open']]: props.animate,
-    [MenuStyles['is-hidden']]: props.hidden,
-
-    // RTL
-    [MenuStyles['is-rtl']]: isRtl(props)
-  })
+  className: props.isAnimated && 'is-animated'
 }))<IStyledMenuViewProps>`
-  && {
-    position: relative;
-    padding: 0;
-  }
+  display: inline-block;
+  position: relative; /* [1] */
+  margin: 0; /* [2] */
+  box-sizing: border-box;
+  border: ${props => `${props.theme.borders.sm} ${getColor('neutralHue', 300, props.theme)}`};
+  border-radius: ${props => props.theme.borderRadii.md};
+  box-shadow: ${props =>
+    props.theme.shadows.lg(
+      `${props.theme.space.base * 5}px`,
+      '30px',
```

Comment:
Ideally this would be `${props.theme.space.base * 7.5}px` (missed it on `dropdowns`), although not super important to change.

Reply:
On the other hand, maybe we want this to be fixed?

Reply:
I vote for keeping this fixed as a base change would could dramatically change a shadow. We may want to look at consolidating these shadows under a themed value/utility?